### ISSUE TITLE
Select dropdown Inputs missing aria-label for accessibility 

### ISSUE THREAD

**Issue by FranciscoVertedor** [catchdigital/drupal-catchify#15]
File (**/docroot/themes/custom/bucks/node_modules/catchify-core/dist/forms.js line 11852 // Add input dropdown**) 
is missing aria-label so screen readers are not able to read these inputs. 

Would it be enough to add `this.input.setAttribute('aria-label', 'select-dropdown`?
